lib/config.js

```diff
@@ -110,6 +110,12 @@ function loadConfig(filePath) {
             // Make the last element in an array take the highest precedence
             config = configExtends.reduceRight(function (previousValue, parentPath) {
 
+                if (parentPath === "eslint:recommended") {
+                    // Add an explicit substitution for eslint:recommended to conf/eslint.json
+                    // this lets us use the eslint.json file as the recommended rules
+                    parentPath = "./conf/eslint.json";
+                }
+
                 if (isFilePath(parentPath)) {
```

Comment:
Correct, good catch. I think this may be the source of a few unexpected test results I was seeing. Thanks for this.